### PR TITLE
minor changes, flow & such

### PR DESCRIPTION
**Pull request by cgreene**
This is a PR into @agitter's branch. I think if this merge is successful it will then get merged (eventually) into his PR into the meta-review. 🤞

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -47,23 +47,23 @@ Although they work well for editing text, they lack sufficient features for mana
...
 Table: Collaborative writing platforms.
 A summary of features that differentiate Manubot from existing collaborative writing platforms.
 We assessed features on June 15, 2018 using the free version of each platform.
 Some platforms offer additional features through a paid subscription or software.
 <sup>1</sup>Additional functionality, such as bibliography management, is available by editing the Word document stored in OneDrive with the paid Word desktop application.
 <sup>2</sup>Conversations about modifications take place on the document as comments, annotations, or unsaved chats.
-There is no separate integrated forum for discussing and editing revisions.
-<sup>3</sup>Simultaneous edits online can be attributed to a single author in a git commit.
+There is no integrated forum for discussing and editing revisions.
+<sup>3</sup>In some circumstances, a git commit containing different edits made at the same time by distinct authors may be attributed to a single author.
```
Comment (dhimmel) [inline content/02.main-text.md:59]: Isn't the Overleaf issue worse? Are we sure any commits are correctly attributed to their author?

I think it should be more like, "git commits created by Overleaf do not correspond to modular revisions and do not retain accurate authorship information."

File: content/02.main-text.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 The internet enables science to be shared in real-time at a low cost to a global audience.
 This development has decreased the barriers to making science open, while supporting new massively collaborative models of research.
-However, the scientific community requires tools whose workflows encourage openness rather than stifle it.
-For example, manuscripts are the cornerstone of scholarly communication, but drafting and publishing manuscripts has traditionally relied on proprietary or offline tools that do not support _open scholarly writing_, where anyone is able to contribute and the contribution history is preserved and public.
-Hence, we report a new tool and workflow for authoring scholarly manuscripts in the open, as well as the collaborative project that led to its creation.
+However, the scientific community requires tools whose workflows encourage openness.
```
Comment (dhimmel) [inline content/02.main-text.md:5]: Is the point still clear without "rather than stifle it"?

**Review by dhimmel — COMMENTED** [review]
Looks good. Two minor comments.

**Comment by dhimmel** [comment]
Assuming this gets merged, we probably want to retain authorship information, but squash commits by both authors. Since https://github.com/greenelab/meta-review/pull/92 is ready to merge (as far as I can tell), maybe it'd be easier to just merge https://github.com/greenelab/meta-review/pull/92 and transfer @cgreene's commits here to a new PR. I can do that if need be.

**Comment by agitter** [comment]
Yes, I please merge my pull request (92) and we can work on this one
independently.

**Comment by dhimmel** [comment]
> Yes, I please merge my pull request (92) and we can work on this one

I am guessing you are telling me to merge 92? Or are you saying you (I in quote) will merge 92.

**Comment by agitter** [comment]
Sorry, bad typo. You (dhimmel) can merge it. I (agitter) won't be able to
immediately.

**Comment by dhimmel** [comment]
Okay merged https://github.com/greenelab/meta-review/pull/92. @cgreene we will coordinate tomorrow on how to base this PR off of `greenelab:master`

**Comment by agitter** [comment]
I'll review and comment on the Overleaf phrasing once we decide whether this pull request is staying here or moving elsewhere.

Would it be easiest to close this and open a new pull request: https://github.com/greenelab/meta-review/compare/master...cgreene:agitter-proofreading

**Comment by agitter** [comment]
The pull request was successfully migrated to https://github.com/greenelab/meta-review/pull/93, and I copied @dhimmel's comments there.

**Pull request closed without merging**